UI/window-basic-settings.cpp

```diff
@@ -3111,6 +3116,14 @@ void OBSBasicSettings::SaveAudioSettings()
 		main->UpdateVolumeControlsDecayRate();
 	}
 
+	if (WidgetChanged(ui->peakMeterType)) {
+		uint32_t peakMeterTypeIdx = ui->peakMeterType->currentIndex();
+		config_set_uint(main->Config(), "Audio", "PeakMeterType",
+			peakMeterTypeIdx);
```

Comment:
Slight indent inconsistency here.  Typically when a function param goes to the next line, there are two indents.  (Mostly just following vim conventions on this).